huaweicloud/config/endpoints.go

```diff
@@ -277,6 +277,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Version: "v1",
 		Product: "SFSTurbo",
 	},
+	"cae": {
+		Name:    "cae",
+		Version: "v1",
+		Product: "CAE",
+	},
```

Comment:
```suggestion
	"cae": {
		Name:    "cae",
		Product: "CAE",
	},
```
Version is useless for auto-generator

Reply:
fixed